src/Hl7.Fhir.Core/Serialization/RepeatingElementReader.cs

```diff
@@ -31,7 +31,7 @@ internal RepeatingElementReader(ISourceNode reader, ParserSettings settings)
 
 #pragma warning restore 612, 618
 
-        public IList Deserialize(PropertyMapping prop, string memberName, IList existing = null)
+        public IList Deserialize(PropertyMapping prop, string memberName, string typeName, IList existing = null)
```

Comment:
This should not need a typeName, since _current.InstanceType already has a type....